prow/plugins/verify-owners/verify-owners.go

```diff
@@ -102,7 +125,18 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, pre *github.Pul
 			modifiedOwnersFiles = append(modifiedOwnersFiles, change)
 		}
 	}
-	if len(modifiedOwnersFiles) == 0 {
+
+	// Check if the OWNERS_ALIASES file was modified.
+	var modifiedOwnerAliasesFile github.PullRequestChange
+	var ownerAliasesModified bool
+	for _, change := range changes {
+		if change.Filename == ownersAliasesFileName {
+			modifiedOwnerAliasesFile = change
+			ownerAliasesModified = true
```

Comment:
You could `break` here? Or even put this inside the `if len(modifiedOwnersFiles) == 0`?
It probably doesn't matter as changes are relatively small anyways...

Reply:
Done in https://github.com/kubernetes/test-infra/pull/12460/commits/2d56cf6d354c07841da7ff5bd0139c3bc4454f81.